packages/driver/test/cypress/integration/commands/querying_spec.coffee

```diff
@@ -1204,6 +1204,15 @@ describe "src/cy/commands/querying", ->
       cy.contains("form", "click me").then ($form) ->
         expect($form.get(0)).to.eq form.get(0)
 
+    it "searches all elements in comma separated filter", ->
+      cy.contains("a,button", "Naruto").then ($el) ->
```

Comment:
will the same work if there are spaces like `cy.contains('a, button')`?

Reply:
I did write a test for this, and it passed, so I removed it...because I thought the same thing and thought maybe we needed to strip the spaces.

But, the more I think about it, I think the test I wrote was likely a false positive...so let me write a better test for this.

Reply:
Wrote a test, trimmed string